src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -12,14 +17,181 @@ const { actions, selectors } = annotations;
  * Create a Redux store which handles annotation, selection and draft actions.
  */
 function createStore() {
-  return createStoreFromModules([annotations, selection, drafts, viewer], [{}]);
+  return createStoreFromModules(
+    [annotations, selection, drafts, groups, session, viewer],
+    [{}]
+  );
 }
 
 // Tests for most of the functionality in reducers/annotations.js are currently
 // in the tests for the whole Redux store
 
-describe('sidebar/store/modules/annotations', function() {
-  describe('isWaitingToAnchorAnnotations', () => {
+describe('sidebar.store.modules.annotations', function() {
```

Comment:
The convention used in the previous code (`sidebar/store/modules/annotations` with slashes) is preferred. We went through a phase of using Python-style module paths in some test descriptions. Please don't use that in new code and feel free to modify existing uses.